app/api/artists/route.tsx

```diff
@@ -12,7 +13,13 @@ export async function GET(req: NextRequest) {
       );
     }
 
-    const artists = await getArtists(accountId);
+    // Parse orgId: "null" string means personal only, undefined means all
+    const parsedOrgId = orgId === "null" ? null : orgId || undefined;
```

Comment:
KISS principle - In what case would orgId be a value of `null` with type `string`?
- actual: advanced string parsing logic makes it possible for edge case bugs to emerge.
- required: directly pass orgId to getArtists and fix any front end cases where a string of `null` is passed.
- Alternatively, use a zod schema to parse the searchParams for more robust / modular searchParams parsing.